assets/queries/dockerfile/copy_from_without_from_alias_defined_previously/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "copy_from_without_from_alias_defined_previously",
+  "queryName": "COPY --from Without FROM Alias Defined Previously",
+  "severity": "MEDIUM",
+  "category": "Multi-stage",
+  "descriptionText": "COPY --from should mention a previously defined FROM alias",
```

Comment:
change to "COPY command with the flag '--from' should mention a previously defined FROM alias"